### DISCUSSION TITLE
Add formatting and linting options into the Makefiles.

### DISCUSSION THREAD

**Discussion by atm1020** [nvim-java/nvim-java#102] [Ideas]
The github actions include the StyLua and LuaCheck tasks, so I believe it would be useful for local development to add linting and formatting options to the makefiles.

for example, adding these two additional options:
```make
lint: 
        luacheck ${SRC_DIR} ${TESTS_DIR}

format:
        stylua ${SRC_DIR}  ${TESTS_DIR} --config-path=.stylua.toml
```

**Comment by s1n7ax** [comment]
Added this: https://github.com/nvim-java/nvim-java/issues/108
